packages/node_modules/@webex/plugin-meetings/src/members/index.js

```diff
@@ -36,58 +40,179 @@ export default class Members extends StatelessSparkPlugin {
    */
   setUpLocusInfoListeners() {
     this.setUpLocusUrlListener();
+    this.setUpLocusHostListener();
+    this.setUpLocusSelfListener();
     this.setUpLocusParticipantsListener();
+    this.setUpLocusMediaSharesListener();
   }
 
   /**
    * @returns {undefined}
    */
-  setUpLocusParticipantsListener() {
-    Events.on('LOCUS_INFO_FULL_PARTICIPANTS', (payload) => {
+  setUpLocusSelfListener() {
```

Comment:
the self nomenclature is hard to read/understand if you aren't familiar with locus. Just a general comment. Maybe some comments helping to explain that if they aren't already elsewhere in the codebase

Reply:
Will add comments, totally agree

Reply:
added comments to members.selfId